NeonShooter/NeonShooter.Android/Content.mgcb

```diff
@@ -0,0 +1,13 @@
+#----------------------------- Global Properties ----------------------------#
+
+/outputDir:bin/$(Platform)
+/intermediateDir:obj/$(Platform)
+/platform:iOS
```

Comment:
should this be Android?

Reply:
Removed all not required default content